series/show.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+import datetime
+
+
+@dataclass
+class Show:
+    """
+    Class to represent the details of a show
+
+    """
+
+    id: int
+    title: str
+    small_logo_path: str
+    big_logo_path: str = None
+    synopsis: str = None
+    creation_date: datetime = None
+    directors: [str] = None
```

Comment:
Does this way of declaring list of strs work? The usual way I see is using `List[str]` (`List` must be imported from the `typings` module).